README.md

```diff
@@ -301,6 +302,45 @@ If you don't have a React application already, start with [create react app](htt
   export default MyCustomComponent;
 ```
 
+### Minimal css best practices
+
+The Discovery Components styles package can be consumed in three ways
+
+1. Wholesale SCSS
```

Comment:
Not married to the definitions here. Just wanted something to quickly describe what the method is. Maybe single import SCSS and partial SCSS imports instead?